web/impact/impact/utils.py

```diff
@@ -45,6 +45,22 @@ def get_profile(user):
         return None
 
 
+def lower_case_model_name(view):
+    model = view.kwargs.get('model', '').lower()
+    related_model = view.kwargs.get('related_model')
+    if related_model:
+        return model + '_' + related_model
+    return model
+
+
+def model_case_model_name(view):
+    model = snake_to_model_name(view.kwargs.get('model', ''))
+    related_model = view.kwargs.get('related_model')
+    if related_model:
+        return model + '_' + related_model
+    return model
```

Comment:
Those two functions are very similar and can probably be merged (note the only difference in `model=...`)

Reply:
Refactored!